app/src/main/res/values/strings.xml

```diff
@@ -128,6 +128,10 @@
     <string name="bigram_prediction">Next-word suggestions</string>
     <!-- Description for "next word suggestion" option. This displays suggestions even when there is no input, based on the previous word. -->
     <string name="bigram_prediction_summary">Use the previous word in making suggestions</string>
+    <!-- Option to enable the suggestion of clipboard content. -->
+    <string name="suggest_clipboard_content">Suggest clipboard content</string>
+    <!-- Description for the "suggest clipboard content" option. This makes the primary clipboard content visible in the suggestion strip view. -->
+    <string name="suggest_clipboard_content_summary">Show the content of the primary clipboard as a suggestion</string>
```

Comment:
Users will not know what a "primary clipboard" is. Maybe call it system clipboard, or just clipboard.
It should also mention that only recently copied items will be in the suggestions.

Reply:
How about `Show recently copied clipboard content as a suggestion`?

Reply:
Yes, that sounds good